pyproject.toml

```diff
@@ -32,8 +32,8 @@ numpy = [
     { version = ">=1.26", python = ">=3.12" },
 ]
 pandas = [
-    { version = ">=1.1,<1.6", python = ">=3.8,<3.11" },
-    { version = ">=1.5,<1.6", python = ">=3.11,<3.12" },
+    { version = ">=1.1,<3", python = ">=3.8,<3.11" },
```

Comment:
We should pin python 3.8 to pandas `>=1.1,<2.1` https://pandas.pydata.org/pandas-docs/version/2.1/whatsnew/v2.1.0.html#increased-minimum-version-for-python

Reply:
Do we need to enforce this or can we just let `pandas` enforce its own compatibility matrix?

Reply:
I wasn't sure if poetry would catch this, but just tested and it does

Reply:
verified with `pdm` with python 3.8 as well

```
pdm add "git+https://github.com/kolenaIO/kolena@gh/relax-pandas"                                                                                                                                                                                                       
Adding packages to default dependencies: git+https://github.com/kolenaIO/kolena@gh/relax-pandas
🔒 Lock successful
Changes are written to pyproject.toml.
Synchronizing working set with resolved packages: 35 to add, 0 to update, 0 to remove

  ✔ Install annotated-types 0.6.0 successful
  ✔ Install dacite 1.8.1 successful
  ✔ Install idna 3.6 successful
  ✔ Install multimethod 1.10 successful
  ✔ Install mypy-extensions 1.0.0 successful
  ✔ Install click 8.1.7 successful
  ✔ Install certifi 2024.2.2 successful
  ✔ Install importlib-metadata 7.1.0 successful
  ✔ Install packaging 24.0 successful
  ✔ Install pandera 0.15.1 successful
  ✔ Install charset-normalizer 3.3.2 successful
  ✔ Install python-dateutil 2.9.0.post0 successful
  ✔ Install pydantic 2.6.4 successful
  ✔ Install requests 2.31.0 successful
  ✔ Install requests-toolbelt 1.0.0 successful
  ✔ Install retrying 1.3.4 successful
  ✔ Install pytz 2024.1 successful
  ✔ Install pillow 10.2.0 successful
  ✔ Install six 1.16.0 successful
  ✔ Install shapely 2.0.3 successful
  ✔ Install tqdm 4.66.2 successful
  ✔ Install termcolor 1.1.0 successful
  ✔ Install typeguard 4.1.5 successful
  ✔ Install typing-inspect 0.9.0 successful
  ✔ Install typing-extensions 4.10.0 successful
  ✔ Install kolena 0.999.0 successful
  ✔ Install urllib3 2.2.1 successful
  ✔ Install zipp 3.18.1 successful
  ✔ Install pydantic-core 2.16.3 successful
  ✔ Install wrapt 1.16.0 successful
  ✔ Install tzdata 2024.1 successful
  ✔ Install pyarrow 15.0.2 successful
  ✔ Install numpy 1.24.4 successful
  ✔ Install pandas 2.0.3 successful
  ✔ Install scipy 1.10.1 successful

🎉 All complete!
```